lang/countries-et.json

```diff
@@ -0,0 +1,255 @@
+{
+  "locale": "et",
```

Comment:
Please drop this file, it's autogenerated.

Reply:
Thanks for the tip. Done!